static/js/core.js

```diff
@@ -1,226 +1,89 @@
  */
 
 if (!core) { var core = {}; }
-YUI().use('node', 'anim', 'event-resize', function (Y) {
-    // REMOVE
-    core.hashSlide = function () {
-        Y.all('#main-content a').each(function (node) {
-            if (node.get('hash') !== '') {
-                var targetURL = node.get('origin') + node.get('pathname');
-                if (window.location.href === targetURL) {
-                    node.on("click", function (e) {
-                        e.preventDefault();
-                        if (!this.hasClass('slideless')) {
-                            core.slideToAnchor(node.get('hash'));
-                        }
-                    });
-                }
-            }
-        });
-    };
-    // REMOVE
-    core.slideToAnchor = function ($name) {
-        if ($name !== '') {
-            var destination = Y.one($name).getXY()[1] - 20,
-                webkitAnim = new Y.Anim({
-                    node: Y.one('html'),
-                    to: { scroll: [0, destination]},
-                    easing: 'easeOut',
-                    duration: 1
-                }),
-                ffAnim = new Y.Anim({
-                    node: Y.one('body'),
-                    to: { scroll: [0, destination]},
-                    easing: 'easeOut',
-                    duration: 1
-                });
-            webkitAnim.run(1000);
-            ffAnim.run(1000);
-        }
-    };
-    // REMOVE
-    core.getPullQuotes = function () {
-        Y.all('span.pullquote').each(function (node) {
-            var item = Y.Node.create('<div class="pull-quote js">&ldquo;' + node.getContent() + '&rdquo;</div>');
-            node.get('parentNode').get('parentNode').get('parentNode').append(item);
-        });
-    };
-    // USING??? IF NOT REMOVE
-    core.setupTooltips = function () {
-        if (Y.one('.tooltip') !== null) {
-            Y.all('.tooltip').each(function (node) {
-                node.get('parentNode').prepend('<p class="tooltip-label">' + node.get('title') + '</p>');
-                var title = this.get('title');
-                node.on('mouseover', function () {
-                    this.set('title', '');
-                    this.get('parentNode').one('.tooltip-label').setStyle('display', 'inline');
-                });
-                node.on('mouseout', function () {
-                    this.set('title', title);
-                    this.get('parentNode').one('.tooltip-label').setStyle('display', 'none');
-                });
-            });
-        }
-    };
-    // REMOVE, UNUSED
-    core.sectionTabs = function () {
 
-        if (Y.one('.tabbed-content')) {
-            Y.one('.tabbed-menu').append('<img src="//assets.ubuntu.com/sites/ubuntu/latest/u/img/patterns/tabbed-nav-arrow.png" class="arrow" height="6" width="12" alt="">');
+core.svgFallback = function() {
+	if (typeof Modernizr === "object") {
+		if (!Modernizr.svg || !Modernizr.backgroundsize) {
+			var svgs = document.querySelectorAll("img[src$='.svg']")
```

Comment:
Oh wait... Actually, adding `?fmt=png` probably won't break most images will it? It will just do nothing.

- <http://s.cdpn.io/3/kiwi.svg?fmt=png>
- <https://www.adobe.com/inspire-apps/exporting-svg-illustrator-0913/fig11/img/napoleon%20for%20svg%201.svg?fmt=png>
- <https://upload.wikimedia.org/wikipedia/en/2/22/Heckert_GNU_white.svg?fmt=png>
- <http://www.freedos.org/images/logos/fdfish-glossy-plain.svg?fmt=png>
- <https://svn.apache.org/repos/asf/flink/img/logo/svg/flink_logotypes.svg?fmt=png>

So don't worry about it.